modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala

```diff
@@ -0,0 +1,264 @@
+package edu.gemini.seqexec.server
+
+import java.util.logging.Logger
+
+import edu.gemini.seqexec.model.dhs.ImageFileId
+import edu.gemini.seqexec.server.GmosSouthController._
+import edu.gemini.spModel.gemini.gmos.GmosCommonType.AmpReadMode
+import edu.gemini.spModel.gemini.gmos.GmosCommonType.AmpGain
+import edu.gemini.spModel.gemini.gmos.GmosCommonType.AmpCount
+import edu.gemini.spModel.gemini.gmos.GmosCommonType.BuiltinROI
+import edu.gemini.spModel.gemini.gmos.GmosCommonType.ROIDescription
+import edu.gemini.spModel.gemini.gmos.GmosCommonType.Order
+import edu.gemini.spModel.gemini.gmos.GmosSouthType.{FilterSouth => Filter}
+import edu.gemini.spModel.gemini.gmos.GmosSouthType.{DisperserSouth => Disperser}
+import edu.gemini.spModel.gemini.gmos.GmosSouthType.{FPUnitSouth => FPU}
+import edu.gemini.spModel.gemini.gmos.GmosSouthType.{StageModeSouth => StageMode}
+
+import squants.Length
+
+import scalaz.Scalaz._
+import scalaz.EitherT
+import scalaz.concurrent.Task
+
+object GmosControllerEpics extends GmosSouthController {
+  private val Log = Logger.getLogger(getClass.getName)
+
+  import EpicsCodex._
+
+  val CC = GmosEpics.instance.configCmd
+  val DC = GmosEpics.instance.configDCCmd
+
+  override def getConfig: SeqAction[GmosSouthConfig] = ???
+
+  implicit val ampReadModeEncoder: EncodeEpicsValue[AmpReadMode, String] = EncodeEpicsValue {
+    case AmpReadMode.SLOW => "SLOW"
+    case AmpReadMode.FAST => "FAST"
+  }
+
+  implicit val shutterStateEncoder: EncodeEpicsValue[ShutterState, String] = EncodeEpicsValue {
+    case OpenShutter  => "OPEN"
+    case CloseShutter => "CLOSED"
+    case _            => ""
```

Comment:
I answered myself. This `shutterStateEncoder` is not used if requested shutter state is not Open or Close.